ibm_catalog.json

```diff
@@ -93,7 +93,366 @@
                 "description": "The Account Infrastructure Base architecture provides a standardized approach to setting up cloud account settings and related audit logging controls as outlined in the IBM Cloud Framework for Financial Services. The architecture includes configurations for IAM settings and access controls that enforce security best practices. Storing Activity Tracker logs in a Cloud Object Storage bucket ensures that the log data is securely retained and easily accessible for analysis, auditing, and compliance purposes. You can use this architecture as a base for deploying additional services and resources in multiple types of accounts, such as administrative or workload accounts."
               }
             ]
-          }
+          },
+          "configuration": [
+            {
+              "key": "ibmcloud_api_key",
+              "type": "password"
```

Comment:
no need to include the type here. It will automatically make it a password type due to the fact the terraform variable is marked as sensitive